js/core/filterMenu.ts

```diff
@@ -460,7 +460,10 @@ export class InteractiveFilterDialog extends BoxPanel {
    */
   protected _evtMouseDown(event: MouseEvent) {
     // Close the menu if a click is detected anywhere else
-    if (!ElementExt.hitTest(this.node, event.clientX, event.clientY)) {
+    if (
+      !ElementExt.hitTest(this.node, event.clientX, event.clientY) &&
```

Comment:
I'm not sure about the logic in this condition. Should it be an OR? Also the condition `!this.node.contains(event.target as HTMLElement)` is probably sufficient.

```suggestion
      !ElementExt.hitTest(this.node, event.clientX, event.clientY) ||
```

Reply:
Thank you!